blog/_posts/2021-05-19-latest-stable.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "New 'latest-stable' branch in git repository"
+layout: archive
+author: Javier Lopez Gomez
+---
+
+This post is a must-read for advanced users that build ROOT from source. Starting with 6.24, we provide a new branch named `latest-stable` that will be regularly updated after each release. If you want to know more, keep reading!
+
+We have listened to you! We had users in the forum that were facing some issues building ROOT's `master` from source. However, in some circumstances the `master` branch might be temporarily broken on some platforms/compilers.
```

Comment:
Might be only me, but I don't see the contradiction that I'm expecting after `however`. Maybe replace by `For example`?